modules/nf-core/spaceranger/count/main.nf

```diff
@@ -0,0 +1,67 @@
+process SPACERANGER_COUNT {
+    tag "$meta.id"
+    label 'process_high'
+
+    // TODO push to nf-core docker
+    container "ghcr.io/grst/spaceranger:2.1.0"
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "SPACERANGER_COUNT module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+
+    input:
+    tuple val(meta), path(reads), path(image), path(alignment), path(slidefile)
+    path(reference)
+    path(probeset)
+
+    output:
+    tuple val(meta), path("**/outs/**"), emit: outs
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def probeset = probeset ? "--probe-set=\"${probeset}\"" : ""
```

Comment:
I could be wrong, but if this is actually empty or `null`, then the process won't run, yes?

Reply:
I guess a similar question would apply for `image`, `alignment`, and `slidefile`

Reply:
These are actually optional inputs. I added some comments to clarify. Is this better now?